### PR TITLE
win_import_emcc_py.

### LINKED ISSUE #3939: Emscripten 1.35.10 with CMake fails to generate system libs, when static libs are .bc
When building an Emscripten tag 1.35.10 app with CMake, with this small change to the toolchain file so that it produce bitcode (.bc) static libs instead of archives (.a)

```cmake
set(CMAKE_STATIC_LIBRARY_SUFFIX ".bc" CACHE STRING "" FORCE)
set(CMAKE_SHARED_LIBRARY_SUFFIX ".bc" CACHE STRING "" FORCE)
set(CMAKE_FIND_LIBRARY_SUFFIXES ".bc" CACHE STRING "" FORCE)
set(CMAKE_C_CREATE_STATIC_LIBRARY "<CMAKE_C_COMPILER> -o <TARGET> <LINK_FLAGS> <OBJECTS>" CACHE STRING "" FORCE)
set(CMAKE_CXX_CREATE_STATIC_LIBRARY "<CMAKE_CXX_COMPILER> -o <TARGET> <LINK_FLAGS> <OBJECTS>" CACHE STRING "" FORCE)
```

...Emscripten gets into an infinite loop when trying to generate system libs...

```
WARNING:root:generating system library: libcxx_noexcept.a...
```

...with the following error:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "D:\emsdk-1.35.0-portable-64bit\python\2.7.5.3_64bit\lib\multiprocessing\forking.py", line 380, in main
    prepare(preparation_data)
  File "D:\emsdk-1.35.0-portable-64bit\python\2.7.5.3_64bit\lib\multiprocessing\forking.py", line 489, in prepare
    file, path_name, etc = imp.find_module(main_name, dirs)
ImportError: No module named em++
```

Small test case to reproduce it : [download here](https://autodesk.box.com/s/ww57p42rwj4wno7z46ewm81mwocr2vg5)

This worked fine in 1.35.4 and lower. I don't know which version it stopped working but we know that in 1.35.10 its broken.

### PR DESCRIPTION
**Pull request by juj**
Fix Windows multiprocessing/forking import of emcc and em++. Windows python 2.7 insists on finding em++.py and does not look up em++, so duplicate the file em++ as em++.py as well. Closes #3939.

### CODE REVIEW

File: em++ [hunk 1]
```diff
@@ -9,8 +9,9 @@ sys.argv += ['--emscripten-cxx']
 
 if sys.version_info.major == 2:
   import emcc
```
Comment (kripken) [inline em++]: why is this import still needed?

File: em++ [hunk 2]
```diff
@@ -9,8 +9,9 @@ sys.argv += ['--emscripten-cxx']
```
Comment (kripken) [inline em++]: we could do the `if __name__ == '__main__'` once on the top level, then put the `if sys.version` inside that? same for the other two scripts.

Unless the `import emcc` has to run even in the non-`__main__` case?

lgtm either way, with that change if possible, or without it if windows won't work with it.

**Comment by juj** [comment]
I believe that the structure of the `import` statements needs to be identical in the code flows when `__name__ == '__main__'` and when not, if one wants to guarantee that scripts are imported in the same order in multiprocessing parent and children, which is why I wrote it in a duplicated way like that. 

I don't know if different ordered imports would ever pose a problem, but wanted to be on the safe side, since the cross-platform differences in python multiprocess spawning are somewhat odd here. I'd like to keep it the structure this way if possible.

**Comment by kripken** [comment]
Agreed, let's be conservative here.

**Comment by kripken** [comment]
Any way to add a test for this fix?

**Comment by juj** [comment]
The only place I see it is when rebuilding the Emscripten cache for system libraries, and we do already invoke the embuilder.py script to do that in parallel, but for some reason, that one doesn't generate the issue on the Windows bot, but I was able to see the issue on my other Windows computer when embuilder.py was being used. I'm not sure if there's something python-version specific here.

**Comment by kripken** [comment]
Ok.

**Pull request merged**